cmd/zpool/zpool_main.c

```diff
@@ -2861,10 +2896,20 @@ print_iostat_header_impl(iostat_cbdata_t *cb, unsigned int force_column_width,
 
 	print_iostat_labels(cb, force_column_width, iostat_top_labels);
 
+	if (cb->cb_kind)
+		printf("      ");
+
+	printf("\n");
+
 	printf("%-*s", namewidth, title);
 
 	print_iostat_labels(cb, force_column_width, iostat_bottom_labels);
 
+	if (cb->cb_kind)
+		printf("  kind");
```

Comment:
> Perhaps if cb_kind is turned into a cb_flag

That might be more consistent.  We're going to need to look in to the best way for these changes to co-exist with the proposed changes in #5368.